src/packages/@support/features/activity-section/layout/ActivitySection.svelte

```diff
@@ -32,6 +35,8 @@
   import { FormatTime } from "@common/utils/formatTime";
   const formatTimeAgo = new FormatTime().formatTimeAgo;
   import ArrowOutward from "@library/icons/ArrowOutward.svelte";
+  import SparrowLogo from "@workspaces/features/rest-explorer/assets/images/sparrow-logo.svelte";
+  import { ImageModal } from "@library/ui/image-modal";
```

Comment:
fix imports

Reply:
you cant import anything here from rest explorer

Reply:
Resolved